### PR TITLE
Log topics page SIMORGH_DATA when number of promos on page does not match number expected from BFF data

### PR DESCRIPTION
**Pull request by LilyL0u**
Sometimes the E2Es fail because the expected number of promos as defined by the number coming from the BFF data does not match the number of promos rendered on the page. Sometimes one (or more) is missing. When checking manually, I can also see a promo missing (see screenshot below where the number expected was 24, but only 23 showed on the page). We don't know why this is, so this logs the page data when the numbers do not match, to get more information and try to find out why.

An example here:
This topic has many pages of items, so it isn’t that there are only 23 available, but the e2es caught it only showing 23 when 24 is expected, and when I manually checked it also showed only 23.

Screenshot 2022-08-25 at 11 13 04

### CODE REVIEW

**Review by RetiredBlob — APPROVED** [review]

**Review by HarveyPeachey — APPROVED** [review]

File: cypress/plugins/index.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ module.exports = (on, config) => {
   on('file:preprocessor', webpackPreprocessor(options));
 
   // eslint-disable-next-line global-require
-  require('cypress-terminal-report/src/installLogsPrinter')(on);
+  require('cypress-terminal-report/src/installLogsPrinter')(on, 10000);
```
Comment (HarveyPeachey) [inline cypress/plugins/index.js]: What does this number argument do? I think it's throwing an error in CI

Comment (LilyL0u) [reply cypress/plugins/index.js]: This was supposed to be the max limit of lines of log but I think I might have done this wrong. Will fix now

Comment (LilyL0u) [reply cypress/plugins/index.js]: (usually default is 800 which isn't enough)

Comment (HarveyPeachey) [reply cypress/plugins/index.js]: Ahh cool. Looks like you just need it as an object with the corresponding key
https://github.com/archfz/cypress-terminal-report#optionsdefaulttrimlength
```suggestion
  require('cypress-terminal-report/src/installLogsPrinter')(on, {defaultTrimLength: 10000});
```

Comment (LilyL0u) [reply cypress/plugins/index.js]: I have done this instead so we can add more options easily

```
 // Add options for the cypress terminal report (cy.logs) here
  const logPrinterOptions = {
    defaultTrimLength: 2000,
  };
  // eslint-disable-next-line global-require
  require('cypress-terminal-report/src/installLogsPrinter')(
    on,
    logPrinterOptions,
  );
```

E2Es pass now, unit test failed instead

**Pull request merged**